bots/c/softcore/Makefile

```
@@ -13,7 +13,7 @@ PLAYER1_BOT_NAME = $(notdir $(PLAYER1_BOT))
 PLAYER2_BOT_NAME = $(notdir $(PLAYER2_BOT))
 
 
-run: stop check_update_configs $(PLAYER1_BOT) $(PLAYER2_BOT)
+run: stop check_updates $(PLAYER1_BOT) $(PLAYER2_BOT)
```

checkmake (0.2.2)

[warning] Target body for "run" exceeds allowed length of 5 (9).

(maxbodylength)